### DISCUSSION TITLE
Logfile: skipped cluster ..., embedding not distinct enough

### DISCUSSION THREAD

**Discussion by ed4711** [photoprism/photoprism#2588] [General]
Hello, first of all. Thanks for photoprism, it is great tool. 

Each time the worker is running I am getting the same message:
 level=info msg="faces: skipped cluster IFB4F6W6AC6SNEUDMDBRTHDYVO24JV37, embedding not distinct enough"
level=info msg="faces: skipped cluster RYMKDKCL6J26A7MH7W5MM2EARYSMMORV, embedding not distinct enough"
 level=info msg="faces: found 2 new clusters"

I tried to search for the ID IFB4F6W6AC6SNEUDMDBRTHDYVO24JV37 or RYMKDKCL6J26A7MH7W5MM2EARYSMMORV via the gui with the following query:
face:IFB4F6W6AC6SNEUDMDBRTHDYVO24JV37
subject:IFB4F6W6AC6SNEUDMDBRTHDYVO24JV37
But i am not getting any result. How can I find these clusters and how can i resolve the issue?

**Comment by lastzero** [comment]
These are skipped based on a blacklist of face embeddings that can not be well recognized based on our tests. Check our code for implementation details. We are currently on vacation, so can't explain it in more detail right now.
